docs/source/index.rst

```diff
@@ -38,6 +39,10 @@ can use it:
    usage: python-build [-h] ...
 
 
+By default python-build will build the package in a virtual environment, but
```

Comment:
considering we're not creating virtual environments it's better to say, by default will build the package in an isolated environment.